README.md

```diff
@@ -31,152 +42,177 @@ Visit the [Ably Asset Tracking](https://ably.com/documentation/asset-tracking) d
+## Resources
+- [Asset Tracking Documentation](https://ably.com/documentation/asset-tracking)
+- [Ably Documentation](https://ably.com/documentation)
 
-### Building / viewing Jazzy documentation locally
-- Run `. ./jazzy/build.sh`
-- Open `docs/index.html`
+## Feature Support
+iOS Client Library [feature support matrix](https://ably.com/documentation/asset-tracking).
 
-### Coding Conventions and Style Guide
+## Support, feedback and troubleshooting
+Please visit http://support.ably.com/ for access to our knowledgebase and to ask for any assistance.
 
-- The SDKs are written in Swift, however they still have to be compatible for use from Objective-C based apps.
-- Favor Protocol Oriented Programming with Dependency Injection when writing any code. We're unable to create automatic mocks in Swift, so it'll be helpful for writing unit tests.
-- SwiftLint is integrated into the project. Make sure that your code does not add any SwiftLint related warning.
-- Please remove default Xcode header comments (with author, license and creation date) as they're not necessary.
-- If you're adding or modifying any part of the public interface of SDK, please also update [QuickHelp](https://developer.apple.com/library/archive/documentation/Xcode/Reference/xcode_markup_formatting_ref/SymbolDocumentation.html#//apple_ref/doc/uid/TP40016497-CH51-SW1) documentation.
-- Docs are written for both Swift and ObjC
+You can also view the [community reported Github issues](https://github.com/ably/ably-asset-tracking-swift/issues).
 
-### Release Procedure
+To see what has changed in recent versions, see the [CHANGELOG](https://github.com/ably/ably-asset-tracking-swift/blob/main/CHANGELOG.md).
 
-#### Bumping the Version
+## Known Limitations
+These SDKs support iOS and iPadOS. Support for macOS / tvOS may be developed in the future, depending on interest / demand.
 
-To increment the version information for each release from the `main` branch:
+## Contributing
 
-1. Search for all instances of the `CFBundleShortVersionString` key in `Info.plist` files and increment according to release requirements, conforming to the [Semantic Versioning Specification](https://semver.org/) version 2.0.0.
-2. Search for all instances of the `CFBundleVersion` key in `Info.plist` files and increment the integer value by 1, ignoring those indirected via `$(CURRENT_PROJECT_VERSION)` (see next step).
-3. Search for all instances of the `CURRENT_PROJECT_VERSION` key in `project.pbxproj` filesand increment the integer value by 1.
+For guidance on how to contribute to this project, see [CONTRIBUTING.md](CONTRIBUTING.md).
 
-The version, both in SemVer string form and as an integer, **MUST** be the same across all projects in this repository (e.g. example app project versions must match those of the library they're built on top of).
+[1]: https://ably.com/
```

Comment:
What are these lines doing? They're not visible in the readme and I can't work out what value they add. We don't do this elsewhere.

Reply:
fixed here 8d9502ef2d7a61158697bf9ea7e5602796959f65